src/Workspaces/Core/Portable/PatternMatching/AllLowerCamelCaseMatcher.cs

```diff
@@ -134,6 +141,21 @@ private PatternMatchKind GetKind(CamelCaseResult result)
                 return bestResult;
             }
 
+            private static char ToLower(char v, TextInfo textInfo)
+            {
+                return IsAscii(v)
```

Comment:
I couldn't find the `IsLatin` check. Also, I'm a bit surprised the virtual overhead would be significant here, considering there aren't any types derived from `TextInfo`. My main holdup here is we aren't considering `IsAsciiCasingSameAsInvariant`, and subtleties in cultures are the things of bug nightmares.